tests/fs/test_mmap.c

```diff
@@ -233,7 +238,7 @@ int main() {
    * is not written beyond the allocated memory area for the mmap operation.
    */
   {
-    int fd = open("/yolo/overallocatedfile.txt", O_RDWR | O_CREAT, (mode_t)0600);
+    int fd = open("./overallocatedfile.txt", O_RDWR | O_CREAT, (mode_t)0600);
```

Comment:
can we instead just replace `/yolo/` with `yolo/` everywhere?    that way the ifdef's around the mkdir won't be needed.

Reply:
Done with commit https://github.com/kleisauke/emscripten/commit/244e4786237d284ac3971949fdbfbf97fc904f12, thanks!